lib/api/src/lib/stories.ts

```diff
@@ -5,16 +5,19 @@ import { dedent } from 'ts-dedent';
 import mapValues from 'lodash/mapValues';
 import countBy from 'lodash/countBy';
 import global from 'global';
-import type {
+import {
```

Comment:
separate types & non-types per @ndelangen's convention

@yannbf is this something we can get a lint rule for?